src/Mjolnir.ts

```diff
@@ -832,6 +834,35 @@ export class Mjolnir {
         await this.printBanlistChanges(changes, banList, true);
     }
 
+    private async handleConsequence(protection: IProtection, roomId: string, event: any, consequence: Consequence) {
+        switch (consequence.type) {
+            case ConsequenceType.alert:
+                break;
+            case ConsequenceType.redact:
+                await this.client.redactEvent(roomId, event["event_id"], "abuse detected");
+                break;
+            case ConsequenceType.ban:
+                await this.client.banUser(event["sender"], roomId, "abuse detected");
```

Comment:
am I doing something wrong here? from testing, this ban doesn't trigger `room.event`, so i'm either banning wrong or listening to the wrong thing